public/app/features/plugins/admin/components/PluginDetailsDisabledError.tsx

```diff
@@ -16,66 +18,119 @@ export function PluginDetailsDisabledError({ className, plugin }: Props): ReactE
-          disabled it. We recommend you to reinstall the plugin to make sure you are running a verified version of this
-          plugin.
+          <Trans i18nKey="plugins.details.disabled-error.invalid-signature-text">
+            Grafana Labs checks each plugin to verify that it has a valid digital signature. While doing this, we
+            discovered that it was invalid. We can not guarantee the trustworthy of this plugin and have therefore
+            disabled it. We recommend you to reinstall the plugin to make sure you are running a verified version of
+            this plugin.
+          </Trans>
         </p>
       );
     case PluginErrorCode.missingSignature:
       return (
         <p>
-          Grafana Labs checks each plugin to verify that it has a valid digital signature. While doing this, we
-          discovered that there is no signature for this plugin. We can not guarantee the trustworthy of this plugin and
-          have therefore disabled it. We recommend you to reinstall the plugin to make sure you are running a verified
-          version of this plugin.
+          <Trans i18nKey="plugins.details.disabled-error.missing-signature-text">
+            Grafana Labs checks each plugin to verify that it has a valid digital signature. While doing this, we
+            discovered that there is no signature for this plugin. We can not guarantee the trustworthy of this plugin
+            and have therefore disabled it. We recommend you to reinstall the plugin to make sure you are running a
+            verified version of this plugin.
+          </Trans>
         </p>
       );
     case PluginErrorCode.failedBackendStart:
-      return <p>This plugin failed to start. Server logs can provide more information.</p>;
+      return (
+        <p>
+          <Trans i18nKey="plugins.details.disabled-error.failed-backend-start-text">
+            This plugin failed to start. Server logs can provide more information.
+          </Trans>
+        </p>
+      );
     case PluginErrorCode.angular:
-      // Error message already rendered by AngularDeprecationPluginNotice
-      return <></>;
+      if (isLatestCompatibleNotAngular) {
+        return (
+          <p>
+            <Trans i18nKey="plugins.details.disabled-error.angular-error-text">
+              The reason this plugin was disabled is that Grafana does not support angular anymore. You can try updating
+              the plugin version to the latest to see if the latest version is working.
+            </Trans>
+          </p>
+        );
+      }
+
+      return (
+        <p>
+          <Trans i18nKey="plugins.details.disabled-error.angular-error-text-no-non-angular-version">
+            The reason this plugin was disabled is that Grafana does not support angular anymore. Unfortunately, the
+            latest version of this plugin is still using angular so you need to wait for the plugin author to migrate
+            away from angular to use this plugin.
```

Comment:
nit:
```suggestion
            This plugin has been disabled as Grafana no longer supports Angular based plugins. Unfortunately, the
            latest version of this plugin still uses Angular so you need to wait for the plugin author to migrate
            to continue using this plugin.
```

Reply:
@sympatheticmoose is it worth pointing users to the [angular deprecation docs](https://grafana.com/docs/grafana/latest/developers/angular_deprecation/angular-plugins/#angularjs-based-plugins) and the table of suggestions regarding angular plugins?

Reply:
@jackw thanks for the review 🙌 , I've added a new link `Read more about angular deprecation` (it's not highlighted in the picture but it's below the "Read more about managing plugins link")